test/uk/gov/hmrc/pillar2submissionapi/controllers/BtnSubmissionControllerSpec.scala

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.pillar2submissionapi.controllers
+
+import play.api.http.Status.CREATED
+import play.api.libs.json.{JsObject, JsValue, Json}
+import play.api.test.FakeRequest
+import play.api.test.Helpers.{defaultAwaitTimeout, status}
+import uk.gov.hmrc.pillar2submissionapi.controllers.BtnSubmissionControllerSpec._
+import uk.gov.hmrc.pillar2submissionapi.controllers.base.ControllerBaseSpec
+import uk.gov.hmrc.pillar2submissionapi.controllers.error.{EmptyRequestBody, InvalidJson}
+
+class BtnSubmissionControllerSpec extends ControllerBaseSpec {
+
+  val btnSubmissionController: BtnSubmissionController = new BtnSubmissionController(cc, identifierAction)
+
+  "BtnSubmissionController" when {
+    "submitBtn() called with a valid request" should {
+      "return 201 CREATED response" in {
+        val result = btnSubmissionController.submitBtn(
+          FakeRequest(method = "", path = "")
```

Comment:
I think `method = "", path = ""` can be removed in all tests. Also consider using a def for result.